cluster/addons/dns/kube2sky/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
```

Comment:
+1 we found that busybox does not have most of the dynamically linked libraries Go uses.